test/Store.test.js

```diff
@@ -227,6 +228,7 @@ describe('Store', function() {
       spy.calledOnce.should.eql(true);
       spy.alwaysCalledWith('', {
```

Comment:
I'll give it a shot, but honestly I'm not really sure where to start.   Maybe if I stare at this test code long enough it will come to me :)